pkg/server/helper.go

```diff
@@ -28,13 +28,13 @@ func GeneratePTS(depBody deploymentPost, org, env string) (v1.PodTemplateSpec, e
 		return v1.PodTemplateSpec{}, errors.New("No URI set")
 	}
 
-	cdir := os.Getenv("POD_CDIR")
-	if cdir == "" {
-		cdir = "10.1.0.0/16"
+	cidr := os.Getenv("POD_CIDR")
+	if cidr == "" {
+		cidr = "10.1.0.0/16"
 	}
 
 	var tempPaths string
-	var intPort int
+	containerPorts := make([]v1.ContainerPort, len(depBody.Paths))
```

Comment:
Line 47 doesn't overwrite it but just assigns a value to the first index. Without the line at 37 we get an invalid memory address error.